### ISSUE TITLE
Fix for Email Button, Added Tooltips on Buttons, Added Active CSS Class to Button Itself, and More

### ISSUE THREAD

**Issue by own3mall** [carrot/share-button#300]
First off, I'm not sure why your repository has a different version of share-button.js than the release does, so I cleaned this up in my repository.  I don't want to create a pull request since it will complicate your branch.  

So, to illustrate my changes, I committed your release version, and then I inserted my changes into it.

https://github.com/own3mall/share-button/commit/1793428f4af63ec7ce44486da6c0cc4c12c9ac88

I fixed the email button not working, added an active CSS class to the share button itself, made it so that clicking on a social media button doesn't toggle the share button close event, and added tooltips to the social media buttons.  Hope this helps if any of my changes might make it into your branch.

Consider this an issue report for all of the things listed above except I've already fixed them in my own repo.

**Comment by PontusTideman** [comment]
Downloaded your file, but the button is NOT displaying in Firefox nor Safari on El Capitan... :/

`<pre><span class="new">new </span><span class="class">share-button</span>();</pre>
  <script src='dist/share-button.js'></script>
  <script>`

The dist-folder is M.I.A also.

**Comment by own3mall** [comment]
Uh what are you talking about?  The code you posted has nothing to do with how you're supported to use the share button... you need to load it Javascript side.

**Comment by own3mall** [comment]
Put this somewhere on the page in the HTML:

`<share-button></share-button>`

Then in a Javascript script:

`new ShareButton({
                optionsHERE
            }); 
`

**Comment by PontusTideman** [comment]
Yes, and none of ´em are working for me! Tried a couple diff solution... Still no share button or network is displayed! Please see my https://github.com/carrot/share-button/issues/302

**Comment by own3mall** [comment]
You're right!  This share-button plugin is poorly written.  I looked through the code, and it is using questionable CSS for calculations and complicated markup when it can be a lot simpler.  As a result, a similar plugin was created which should provide the same functionality share-button provides in the form of a much simpler jQuery plugin.

Please see here:

https://github.com/earnolmartin/jquery_social_media_share_plugin

Could you help me test this version?  It should be in a decent state at the moment, and it works in IE9-11+, Firefox, Chrome, and Safari.
